test/plotrecipes.jl

```diff
@@ -378,3 +378,33 @@ end
     # fig, ax, _ = M.volumeslices(A3abc; z=:a)
     # M.volumeslices!(ax, A3abc;z=:a)
 end
+
+@testset "AlgebraOfGraphics" begin
+    using AlgebraOfGraphics, CairoMakie
+    using DimensionalData
+
+    # 1d
+    A1 = rand(X(1:5); name=:test)
+    A1c = rand(X('a':'e'); name=:test)
+
+    @testset "1d, symbol indexing" begin
+        @test_nowarn data(A1) * mapping(:X, :test) * visual(CairoMakie.Lines) |> draw
+        @test_nowarn data(A1c) * mapping(:X, :test) * visual(CairoMakie.Lines) |> draw
+    end
+
+    @testset "1d, dim indexing" begin
+        @test_nowarn data(A1) * mapping(X, Dim{:test}) * visual(CairoMakie.Lines) |> draw
```

Comment:
The test error here is because I'm passing in `Dim{:test}`.  If I don't, this is fine. 

Do we care about this syntax?  I almost want to remove it, since the data name isn't technically a dimension of the array.  At the same time, for an n-dimensional DimArray, the data is the n+1th dimension...

Reply:
That shouldn't work right? It's not a dimension